MVC/Model/model_hints_module.py

```diff
@@ -0,0 +1,122 @@
+from puzzle import *
+
+'''    
+m_letters = puzzle.pangram
+m_word_count = 0
+m_points = puzzle.total_points
+m_pangram_count = 0
+m_letterMatrix = [[str()]]
+m_twoLetterDictionary = {}
+#puzzle = puzzle.stats
+m_words =  puzzle.current_word_list'''
+
+
+# Returns string of letters for the current puzzle
+def generateLetters():
+    return puzzle.pangram
+# Returns string of word count
+def generateWordCount(words):
+    words = puzzle.current_word_list
+    word_count = 0
+    for word in words:
+        word_count += 1
+    return str(word_count)
```

Comment:
Python has a 'len()' function to return the size of a list. You can directly return the length of 'words' with this.